### PR TITLE
Minor issue in overlaps function

### PR DESCRIPTION
**Pull request by lisawink**
Minor fix to issue https://github.com/sjsrey/geoplanar/issues/58

### CODE REVIEW

**Review by martinfleis — COMMENTED** [review]
I'd just remove the `.T` to make it work. 

@sjsrey is there a reason to return list here? If not, we can probably directly return the output of `query` with no changes.

**Comment by sjsrey** [comment]
Since I neglected to document the original function, I can't say why a list was returned. 
The only place we are using that function is: https://github.com/sjsrey/geoplanar/blob/103d35488ed15a43092af129a8d5a042a2c8f07f/geoplanar/planar.py#L213

But that  is not directly being tested.

**Comment by martinfleis** [comment]
> directly return the output of query with no changes.

Let's do this then. @lisawink can you update the PR?

**Comment by lisawink** [comment]
@martinfleis sure! is this what you mean?

**Comment by martinfleis** [comment]
Just return the output directly, no need to split it to `i, j`

File: geoplanar/overlap.py [hunk 1]
```diff
@@ -18,12 +18,17 @@
...
 
 def overlaps(gdf):
-    if GPD_GE_014:
-        i, j = gdf.sindex.query(gdf.geometry, predicate="overlaps").T
-    else:
-        i, j = gdf.sindex.query_bulk(gdf.geometry, predicate="overlaps").T
+    """Check for overlapping geometries in the GeoDataFrame.
+
+    Parameters:
+    gdf: The input GeoDataFrame.
```
Comment (martinfleis) [inline geoplanar/overlap.py:24]:
```suggestion
    Parameters
    ----------
    gdf:  GeoDataFrame with polygon geometries
```

proper formatting

**Pull request merged**